code/modules/recycling/disposal.dm

```diff
@@ -242,6 +242,9 @@
 
 /obj/machinery/disposal/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1)
 	var/data[0]
+	
+	if(!air_contents)
+		air_contents = new()	

```

Comment:
This can be left out; pretty sure I know why the gas is disappearing, and will make a PR for it once I've tested the fix.